sourmash/__main__.py

```diff
@@ -12,6 +12,13 @@
                        plot, watch, info, storage, migrate)
 from .lca import main as lca_main
 
+try:
+    from sourmash_utils.__main__ import main as utils_main
+    UTILS_AVAILABLE = True
+except ImportError:
+    UTILS_AVAILABLE = False
```

Comment:
This is here because the original PR was against #236 and I changed it to be against master to have Travis run the tests...